osu.Desktop/Windows/WindowsKey.cs

```diff
@@ -14,6 +14,8 @@ internal class WindowsKey
 
         private const int wh_keyboard_ll = 13;
         private const int wm_keydown = 256;
+
+        // ReSharper disable once IdentifierTypo
```

Comment:
Not sure how others feel but I think that for platform-native classes like this it's better to just disable typo inspections file-wide rather than play whack-a-mole locally, as they generally tend to have obscure names from native APIs that may spuriously trigger the inspection at some point.

Reply:
Agree